openproblems/tasks/chromatin_potential/__init__.py

```diff
@@ -1,7 +1,7 @@
 from ... import utils
 from . import datasets, methods, metrics, checks
 
-_task_name = "Chromatin accessibility prediction"
+_task_name = "Gene expression prediction"
```

Comment:
@dburkhardt thoughts on this task name?

Reply:
Now the name is Regulatory effect prediction.